src/pcrlock/pcrlock.c

```diff
@@ -4836,6 +4846,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  list-components             List defined .pcrlock components\n"
                "  predict                     Predict PCR values\n"
                "  make-policy                 Predict PCR values and generate TPM2 policy from it\n"
+               "  can-make-policy             Check whether the TPM supports PolicyAuthorizeNV\n"
```

Comment:
hmm, would prefer a more generic name for this, maybe just "is-supported" or so.

And please drop the reference to PolicyAuthorizeNV. That's mostly an implementation detail we shouldn't throw at users.

I think we should add a more complex check behind this, and have the freedom to add even more checks later on, if we need more.

Reply:
ack